sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -309,6 +309,44 @@ You should see output similar to:
     node.js version     :  v6.10.3
     npm version         :  3.10.10
 
+The next check that can be made is to run some of the examples. There are
+different ways in which bokeh can be used which suit a variety of use cases.
+
+To create an html file,
+
+.. code-block:: sh
+
+    BOKEH_RESOURCES=inline python examples/plotting/file/iris.py
+
+which will create a file ``iris.html`` locally and open up a web browser.
+
+.. image:: /_images/bokeh_iris_html.png
+    :scale: 50 %
+    :align: center
+
+The variable ``BOKEH_RESOURCES`` determines where the css and JavaScript
+resources required by bokeh are found. In this case they are included inline in
+the html file. The ``BOKEH_RESOURCES`` variable is required as the default
+behaviour is to use CDN resources.
+`resources reference`_.
```

Comment:
I think this is an extra line? Also I think it is specifically worth mentioning that the reason `BOKEH_RESOURCES` is needed is because we want to use the local version of BokehJS that we just built.